corrector.rb

```diff
@@ -0,0 +1,7 @@
+class Corrector
+  def correct_name(name)
+    corrected_name = name.capitalize
+    corrected_name = corrected_name[0..9] if name.length > 10
+    corrected_name
+  end
+end
```

Comment:
- [ ] Kindly note that you need to follow very well the [requirement of this project ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md) by :point_down: 
- Creating `Namable` class with a method 'correct_name' raising an error. 
-  Checking if the `Decorator` class inherits from Namable.
-  Checking if in the constructor a namable object from params is assigned to an instance variable.
-  Checking of the correct_name method that returns the result of the correct_name method of the @namable is implemented.
- Creating a class that inherits from the base Decorator class.
- Implementing a method correct_name that capitalizes the output of @nameable.correct_name.
- Creating a class that inherits from the base Decorator class.
- Implementing a method correct_name that makes sure that the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word.

#### Kindly make sure everything is done in the :point_right: [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md) before requesting the next review. Thanks.